pkg/splunk/enterprise/afwscheduler_test.go

```diff
@@ -25,7 +26,7 @@ import (
 	"testing"
 	"time"
 
-	enterpriseApi "github.com/splunk/splunk-operator/api/v3"
+	enterpriseApiV3 "github.com/splunk/splunk-operator/api/v3"
```

Comment:
Do we need a backlog story item, to cover the clusterManager as well? Starting this release, if my understanding is correct, we are enforcing the customers to use ClusterManager(?), if so, we need to cover all the test scenarios as soon as possible?

@gaurav-splunk , @vivekr-splunk

Reply:
The tests for *Manager were added in separate files that are similar to their *Master counterparts. For example, we created `test/m4/appframework/manager_appframework_test.go` for Manager, while keeping the `original test/m4/appframework/appframework_test.go` for Master.

Reply:
any new creation in V4 we are asking user to use V4, if they are doing it in V3 we still support. yes in V4 having the 2 API to serve same resource do not make sense.